src/app/game-state/home.service.ts

```diff
@@ -767,7 +767,7 @@ export class HomeService {
       if (this.fields[i].daysToHarvest <= 0){
         let fieldYield = this.fields[i].yield;
         if (this.fields.length + this.extraFields > 300){
-          fieldYield = Math.floor((this.fields.length + this.extraFields) / 300);
+          fieldYield = Math.floor(fieldYield * (this.fields.length + this.extraFields) / 300);
```

Comment:
Is this right? This seems funny. @Hornsoup , second opinion please.

Reply:
Yeah, as is the fieldYield is completely overwritten, it should be multiplied by the factor of extrafields, so say you have 600 fields, right now this just sets the fieldYield to 2, instead of 2 * yield.

Reply:
Basically ignores all farming with 301+ fields.